lib/chef/provider/cron.rb

```diff
@@ -221,7 +222,18 @@ def get_crontab_entry
           newcron << "#{v.to_s.upcase}=\"#{new_resource.send(v)}\"\n" if new_resource.send(v)
         end
         new_resource.environment.each do |name, value|
-          newcron << "#{name}=#{value}\n"
+          if ENVIRONMENT_PROPERTIES.include?(name)
+            unless new_resource.property_is_set?(name.downcase)
+              logger.warn("#{new_resource.name}: the environment property contains the '#{name}' variable, which should be set separately as a property.")
+              new_resource.send(name.downcase.to_sym, value.gsub(/^"|"$/, ""))
+              new_resource.environment.delete(name)
+              newcron << "#{name.to_s.upcase}=\"#{value}\"\n"
```

Comment:
why the addition of the double quotes around value? Those weren't present before. That would trigger an update for everyone?

Reply:
Presumably to match line 222?

Reply:
I think, may be to support spaces in values ?
Like in PATH where some directory name could contain spaces.